app/app-config/segmentsConfig.js

```diff
@@ -0,0 +1,35 @@
+define([], function() {
+/* eslint-enable max-statements */
+
+    var segmentData = [
+        {
+            id: 'bikes',
+            url: '/',
+            items: [
```

Comment:
Not sure about this one; all the methods on `SegmentedPlugin` do use "item" in their wording, eg. `setItems`, `selectItem`... so using "items" here is consistent with that